curated_transformers/models/bert/encoder.py

```diff
@@ -41,10 +41,7 @@ def forward(
         Shapes:
             input_ids, token_type_ids - (batch, seq_len)
 
-        `attention_mask` indicates elements to attend to with `True` (and `False` otherwise)
-
-        Returns a tuple of consisting of a list of tensors from each Transformer
-        layer and the sum of the input and positional embeddings.
+        `attention_mask` indicates elements to attend to with `True` (and `False` otherwise).
```

Comment:
We should leave the true/false part out here, since we now have a dedicated class.